doc/includes/reveal/examples_reveal_ajax.html

```diff
@@ -1,10 +1,10 @@
 {{#markdown}}
 ```html
-<button data-reveal-id="myModal" data-reveal-ajax="http://some-url">
+<button data-reveal-id="myModal" data-reveal-ajax="http://some-url" aria-haspopup="true">
     Click Me For A Modal
 </button>
 
-<a href="http://some-url" data-reveal-id="myModal" data-reveal-ajax="true">
+<a href="http://some-url" data-reveal-id="myModal" data-reveal-ajax="true" aria-haspopup="true" role="button">
```

Comment:
This should be a `<button>` tag. `aria-haspopup` and `role="button"` not needed.